lib/api/mapBenefits.js

```diff
@@ -67,7 +68,9 @@ export function MapEICard(eiBenefitData) {
       break
 
     case StatusCodes.applicationReceived:
-      const estimatedDateOfDecision = new Date(eiBenefitData.nextRptDueDate)
+      const estimatedDateOfDecision = new Date(
+        formatDate(eiBenefitData.publishDate, 'en')
```

Comment:
To be clear, I was doing this to add the `T12:00:00.000Z` to the date and then parsing it, to avoid inconsistencies with the shown date. Though, it now occurs to me that there's a better way to go about this, so I'll do that.